src/QRgen/renderer.nim

```diff
@@ -47,6 +47,7 @@ proc renderImg*(
   moRad: Percentage = 0,
   moSep: Percentage = 0,
   pixels: uint32 = 512,
+  padding: uint8 = 5,
   img: Image = Image(width: 0, height: 0),
   imgCoords: tuple[x, y, w, h: uint8] = self.genDefaultCoords
```

Comment:
I think it would be better if you made the new parameter the last one, so that backwards compatibility remains, although I actually like it more were you left it.

Reply:
That makes sense, moved it to the end.